### PR TITLE
ADEN-8646 Get rid of context's listeners.*

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Removes also:
- Twitch player and all related code
- slotListener
- different ways of emitting ad-slot related events (adSlot vs eventService), from now all events going to be dispatched directly on adSlot

### CODE REVIEW

File: platforms/gamepedia/setup-ad-engine.ts [hunk 1]
```diff
@@ -62,10 +63,8 @@ function startAdEngine(): void {
 		}
 	});
 
-	context.push('listeners.slot', {
-		onRenderEnded: (slot) => {
-			slot.getElement().classList.remove('default-height');
-		},
+	eventService.on(AdSlot.SLOT_RENDERED_EVENT, (slot) => {
+		slot.getElement().classList.remove('default-height');
```
Comment (jacekbj) [inline platforms/gamepedia/setup-ad-engine.ts]: Please use `slot.removeClass`.

File: platforms/sports/setup-ad-engine.ts [hunk 2]
```diff
@@ -34,10 +34,8 @@ function startAdEngine(): void {
 	engine.init();
 	babDetection.run();
 
-	context.push('listeners.slot', {
-		onRenderEnded: (slot) => {
-			slot.getElement().classList.remove('default-height');
-		},
+	eventService.on(AdSlot.SLOT_RENDERED_EVENT, (slot) => {
+		slot.getElement().classList.remove('default-height');
```
Comment (jacekbj) [inline platforms/sports/setup-ad-engine.ts]: ditto

Comment (fraszczakszymon) [reply platforms/sports/setup-ad-engine.ts]: :+1:

File: platforms/gamepedia/setup-ad-engine.ts (hunk 1 above)
Comment (fraszczakszymon) [reply platforms/gamepedia/setup-ad-engine.ts]: :+1:

**Pull request merged**